.jshintignore

```diff
@@ -80,6 +80,8 @@ tv_apps/smart-home/bower_components/**
 tv_apps/smart-settings/bower_components/**
 tv_apps/smart-system/bower_components/**
 tv_apps/notification/bower_components/**
+tv_apps/fling-tutorial/bower_components/**
+tv_apps/fling-tutorial/offline_content/bower_components/**
```

Comment:
I would prefer more to put directories like
`js/offline_content`
`style/offline_content`